### PR TITLE
add got favicon for #19

### PR DESCRIPTION
**Pull request by georgettica**
I have not much of frontend expertise, so I just got the favicons from http://www.gameofthrones.com/ and added them to the html page.

Please tell me how I can make this a perfect PR

### CODE REVIEW

**Review by martin-gnodtke — COMMENTED** [review]
First thing its not good to just steal images from other websites to use them as a favicon here. I dont think gameofthrones would be happy about this. :open_mouth:

And second your code is not working, because your icons are linked to an folder which doesnt exist in this project. If you change your links to the asset folder, where the favicon images are located, it should work. But better practise is to put all favicon files into the root directory where index.html and style.css, etc. is located. 

Happy Coding :smiley:

File: index.html [hunk 1]
```diff
@@ -10,6 +10,12 @@
   <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no,user-scalable=0">
   <link rel="stylesheet" href="style.css">
   <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+  <link rel="shortcut icon" href="/favicon.png"/>
+  <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico">
+  <link rel="icon" href="/images/favicon-16.png" sizes="16x16">
```
Comment (itaditya) [inline index.html]:
```suggestion
  <link rel="icon" href="/assets/favicon-16.png" sizes="16x16">
```

this is the actual path. please correct everywhere.

File: index.html [hunk 2]
```diff
@@ -10,6 +10,12 @@
   <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no,user-scalable=0">
   <link rel="stylesheet" href="style.css">
   <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+  <link rel="shortcut icon" href="/favicon.png"/>
```
Comment (itaditya) [inline index.html]: there is no favicon.png file, you should add one

**Comment by martin-gnodtke** [comment]
android-icon-96x96

I created a alternative favicon, if we shouldnt use the original favicon from gameofthrones.com
Let me know if you want to use it, then i open a new PR.

**Comment by itaditya** [comment]
old logo is better

**Comment by georgettica** [comment]
Thank you. I am still searching for a good HQ photo.
I'll fix the links soon

**Comment by georgettica** [comment]
I tried adding the icons, but they are not showing..

how should I change the html or any of the code to support it?

**Comment by georgettica** [comment]
I'm closing thie PR, for I didn't fix anything here.. If you want my resources tell me

**Pull request closed without merging**